sunpy/spectra/spectrum.py

```diff
@@ -36,10 +36,10 @@ def __new__(cls, data, *args, **kwargs):
         return np.asarray(data).view(cls)
 
     def __init__(self, data, freq_axis):
-        if np.shape(data)[0] != np.shape(freq_axis)[0]:
-  	        raise ValueError('Dimensions of data and Frequency axis do not match')
+        if np.shape(data)[0]!= np.shape(freq_axis)[0]:
+            raise ValueError('Dimensions of data and Frequency axis do not match')
         else:
```

Comment:
Yes @wafels . I can remove it if you want me to. I would think as it's either an 'error or 'self.freq_axis=freq_axis', hence kept in two different blocks. Let me know if I should remove the "else: " block